docs/source/actions/merge.rst

```diff
@@ -171,6 +173,20 @@ request:
 Defining the Commit Message
 ---------------------------
 
+|deprecated tag|
+
+.. warning::
+
+    ``commit_message`` is deprecated and will be removed on April 25th, 2022. It's replaced by :ref:`commit_message_template <commit_message_template>`.
+    To migrate from ``commit_message: title+body`` to the :ref:`commit_message_template <commit_message_template>` attribute, set it to:
+
+
+   \```
+   {{ title }} (#{{ number }})
+   
+   {{ body }}
+    \```
+
```

Comment:
quite ugly https://docs-preview.mergify.com/3447/docs/docs/actions/merge/#defining-the-commit-message 😢

Reply:
will it be better if i put it in a code-block ?

Reply:
try it?